Sources/PopupBridge/POPPopupBridge.swift

```diff
@@ -76,10 +84,12 @@ public class POPPopupBridge: NSObject, WKScriptMessageHandler {
 
         if let payloadData = try? JSONEncoder().encode(payload),
            let payload = String(data: payloadData, encoding: .utf8) {
+            Self.analyticsService.sendAnalyticsEvent(PopupBridgeAnalytics.succeeded, sessionID: sessionID)
```

Comment:
Take it or leave it, we could make `sendAnalyticsEvent` take in a `PopupBridgeAnalytics` as the first argument, then just use:
```suggestion
            Self.analyticsService.sendAnalyticsEvent(.succeeded, sessionID: sessionID)
```

Reply:
Ah you're right, forgot these were caseless enums.